doc/source/news/14.md

```diff
@@ -1,5 +1,110 @@
 # News - 14 series
 
+(release-14-0-7)=
+## Release 14.0.7 - 2024-08-29
+
+### Improvements
+
+* [NormalizerNFKC] Added a new option `unify_alphabet_diacritical_mark`.[pgroonga/pgroonga#418][Reported by Man Hoang]
+
+  This option enables that alphabets with diacritical mark and alphabets without diacritical mark regarded as the same character as below.
+
+  ```
+  table_create --name Contents --flags TABLE_HASH_KEY --key_type ShortText
+  column_create --table Contents --name content --type ShortText
+  load --table Site
+  [
+    {"_key":"1", "content":"con đường ngoằn nghoèo"},
+  ]
+
+  table_create \
+    --name Terms \
+    --flags TABLE_PAT_KEY \
+    --key_type ShortText \
+    --default_tokenizer TokenBigram \
+    --normalizer NormalizerNFKC150("unify_alphabet_diacritical_mark", true)
+  column_create \
+    --table Terms \
+    --name index_content \
+    --flags COLUMN_INDEX|WITH_POSITION \
+    --type Contents \
+    --source content
+
+  select --table Contents --query content:@ngoan
+  [
+    [
+      0,
+      0.0,
+      0.0
+    ],
+    [
+      [
+        [
+          1
+        ],
+        [
+          [
+            "_id",
+            "UInt32"
+          ],
+          [
+            "_key",
+            "ShortText"
+          ],
+          [
+            "content",
+            "ShortText"
+          ]
+        ],
+        [
+          1,
+          "1",
+          "con đường ngoằn nghoèo"
+        ]
+      ]
+    ]
+  ]
+  ```
+
+* Added support for simdjson.
+
+  This keeps RapidJSON support but we'll remove RapidJSON support later.
```

Comment:
How about adding the link to `simdjson` repository or official web page?
- https://github.com/simdjson/simdjson
- https://simdjson.org/